docs/rum_collection/data_collected.md

```diff
@@ -9,13 +9,26 @@ further_reading:
     tag: "Documentation"
     text: "Datadog Real User Monitoring"
 ---
-The RUM SDK generates events that have associated metrics and attributes. Metrics are quantifiable values that can be used for measurements related to the event. Attributes are non-quantifiable values used to slice metrics data (group by) in analytics.
+
+## Overview
+
+The RUM iOS SDK generates events that have associated metrics and attributes. Metrics are quantifiable values that can be used for measurements related to the event. Attributes are non-quantifiable values used to slice metrics data (group by) in analytics. 
 
 Every RUM event has all of the [default attributes](#default-attributes), for example, the device type (`device.type`) and user information such as their name (`usr.name`) and their country (`geo.country`).
 
 There are additional [metrics and attributes that are specific to a given event type](#event-specific-metrics-and-attributes). For example, the metric `view.time_spent` is associated with "view" events and the attribute `resource.method` is associated with "resource" events.
 
-This page provides descriptions of each of the metrics and attributes collected.
+| Event Type | Retention | Description                         |
+|------------|-----------|-------------------------------------|
+| Session    | 30 days   | A session represents a real user journey on your mobile application. It begins when the user launches the application, and the session remains live as long as the user stays active. During the user journey, all RUM events generated as part of the session will share the same `session.id` attribute. |
```

Comment:
We might want to add a note that the session is reset after 15 min of inactivity, and can be reset when the app is in the background if it gets killed by the OS

Reply:
The 15 minutes inactivity yes, but the "app being killed" only applies to mobile